module_utils/rho_cmd.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2009-2017 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+"""RhoCmd: a base class with common functionality for our Ansible modules."""
+
+import subprocess as sp
```

Comment:
I strongly agree about not liking the renaming. I left it to keep this PR simpler, but I would like to go back and change it in the future.